arch/RISCV/RISCVInstPrinter.c

```diff
@@ -0,0 +1,148 @@
+//===-- RISCVInstPrinter.cpp - Convert RISCV MCInst to asm syntax ---------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+//
+// This class prints an RISCV MCInst to a .s file.
+//
+//===----------------------------------------------------------------------===//
+
+#ifdef CAPSTONE_HAS_RISCV
+
+#include <stdio.h>		// DEBUG
+#include <stdlib.h>
+#include <string.h>
+#include <capstone/platform.h>
+
+#include "RISCVInstPrinter.h"
+#include "RISCVBaseInfo.h"
+#include "../../MCInst.h"
+#include "../../SStream.h"
+#include "../../MCRegisterInfo.h"
+#include "../../utils.h"
+#include "RISCVMapping.h"
+
+//#include "RISCVDisassembler.h"
+
+#define GET_REGINFO_ENUM
+#include "RISCVGenRegisterInfo.inc"
+#define GET_INSTRINFO_ENUM
+#include "RISCVGenInstrInfo.inc"
+
+static void printRegName(SStream * OS, unsigned RegNo);
```

Comment:
please remove the space between `*` and a variable next to it.
there are still many places with this issue.